File: cpp/prostokąt.cpp
/*
 * prostokąt.cpp
 */

using namespace std;

int obwod(int a, int b)
{
    return 2 * a + 2 * b;
}

int pole(int a, int b)
{
    return a * b;
}

#include <iostream>

int main(int argc, char **argv)
{
    int a, b;
    a = b = 0;
    
    cout << "Podaj pierwszy blok: ";
    cin >> a;
    
    cout << "Podaj drugi blok: ";
    cin >> b;
    
    cout << a << " " << b;
    
    cout << endl << "Obwód prostokąta: " << obwod(a, b) << endl;
    cout << endl << "Pole prostokąta: " << pole(a, b) << endl;

    return 0;
}

File: cpp/prostokąt.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  prostokat.py


def obwod(a, b):
    return 2 *a + 2 * b

def pole(a, b):
    return a * b
    
    
def main(args):
    a = int(input("Podaj pierwszy bok: "))
    b = int(input("Podaj drugi bok: "))
    print(a, b)
    
    print("Obwód prostokąta: ", obwod(a, b))
    print("Pole prostokąta: ", pole(a, b))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
